addons/vagrant/inventory/group_vars/linux_servers/samba4ad.yml

```diff
@@ -0,0 +1,40 @@
+---
+samba__base_packages:
+  - 'samba'
+  - 'winbind'
+  - 'libnss-winbind'
+  - 'krb5-user'
+  - 'smbclient'
+  - 'ldb-tools'
+  - 'python3-cryptography'
+
+samba__services_to_stop:
+  - 'samba-ad-dc.service'
+  - 'smbd.service'
+  - 'nmbd.service'
+  - 'winbind.service'
+  #- 'systemd-resolved'
+
+samba__global_custom:
+  netbios_name: DC1
+  workgroup: EXAMPLE
+
+domain_setup__username: administrator
+domain_setup__password: VagrantPass1
+domain_setup__kdc: yes
+domain_setup__realm: false
```

Comment:
These variables must be renamed to from `domain_setup__` to `samba__` because we don't use that role anymore.

Reply:
Hum, remember to change it in ./addons/vagrant/playbooks/linux_servers/samba4ad.yml
and use samba4ad instead of samba

Reply:
done